resources/js/copy_clipboard.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file
+ * Attaches behaviors to copy text to clipboard for elements with data-copy-target attribute.
+ */
+(function (Drupal) {
+
+  /**
+   * Attaches the copy-to-clipboard behavior to elements with the 'data-copy-target' attribute.
+   *
+   * @type {Drupal~behavior}
+   *
+   * @prop {Drupal~behaviorAttach} attach
+   *   Initializes copy-to-clipboard functionality for each element.
+   * @prop {Drupal~behaviorDetach} detach
+   *   Cleans up any event listeners or instances to avoid memory leaks.
+   */
+  Drupal.behaviors.copyClipboard = {
+    attach: function (context) {
+      Array.prototype.forEach.call(document.querySelectorAll('[data-copy-target]'), function (element) {
+        var targetClass = element.getAttribute('data-copy-target');
+        var targetElement = document.querySelector('.' + targetClass);
```

Comment:
One big problem here: if there are two copyright elements with the same class, the first one in the page will be always copied. If we want to be similar to bootstrap, let's just use the whole selector for the query, and leave the choice to the user. Otherwise we need to make sure to use an ID. Maybe ID in this case is better, or users won't know and pass a class and encounter the same issue.

Reply:
changed.. now it has the whole selector as attr